Misc/NEWS.d/next/Tests/2020-07-01-17-40-32.bpo-41100.lGf8DF.rst

```diff
@@ -0,0 +1 @@
+test fixes for Mac OS 11
```

Comment:
I don't know if a NEWS is necessary for tests fixes. But I would like to wait for a core developer opinion

Reply:
The bot made me do it.   I don't think i have permission to add the label necessary to skip news.

Reply:
@lawrence-danna-apple I added the label for skip news.